File: test_data/code/java/1742-maximum-number-of-balls-in-a-box.java
class Solution {
    public int countBalls(int lowLimit, int highLimit) {
        int[] count = new int[76]; // boxes from 1 to 75
        for (int i = lowLimit; i <= highLimit; i++) {
            int sum = 0;
            int j = i;
            while (j != 0) {
                sum += j % 10;
                j /= 10;
            }
            count[sum]++;
        }
        int max = 0;
        for (int i = 1; i < count.length; i++) {
            if (count[i] > max) {
                max = count[i];
            }
        }
        return max;
    }
}

File: test_data/code/python/1742-maximum-number-of-balls-in-a-box.py
class Solution:
    def countBalls(self, lowLimit: int, highLimit: int) -> int:
        box_counts = {}
        for i in range(lowLimit, highLimit + 1):
            digit_sum = sum(int(digit) for digit in str(i))
            box_counts[digit_sum] = box_counts.get(digit_sum, 0) + 1
        return max(box_counts.values())